docs/pom.xml

```diff
@@ -260,7 +260,6 @@
                         </goals>
                         <configuration>
                             <outputDirectory>${project.build.directory}/generated-docs/apidocs</outputDirectory>
-                            <failOnError>true</failOnError>
```

Comment:
I'm a little concerned removing this line means syntax errors in the docs won't cause a compile error, but instead generate bad docs

Reply:
removing this line makes it fall back to `<failOnError>${basepom.check.fail-javadoc}</failOnError>` in the base pom. That is enforced for the release (in the oss profile), so the build will still fail.

We can add this switch to the doc build as well so that the CI doc build would fail.

Reply:
Ah, great.

Reply:
Yeah, let's make sure the CI doc build fails if the docs don't build - that way we cannot merge it until we fix it, rather than only figuring it out at release.